CMakeLists.txt

```diff
@@ -135,13 +135,13 @@ else()
     set(CMAKE_POSITION_INDEPENDENT_CODE ON)
 endif()
 
-add_library(avif)
+add_library(avif_obj OBJECT)
 
-# Adds <target> to avif's public link libraries for build, and adds
+# Adds <target> to avif_obj's public link libraries for build, and adds
```

Comment:
It seems that the `avif_target_link_library` function (line 143) should operate on the avif library rather than the avif_obj library.

For example, this function talks about installation, but we don't install the avif_obj library.

Reply:
Both would work as they get transmitted transitively. Still, because of the install, I switch to `avif` for the `INSTALL_INTERFACE`